src/app/courses/courses-filter/courses-filter.component.html

```diff
@@ -1,7 +1,18 @@
 <form class="form-group mt-2">
-<div class=" d-flex justify-content-center">
-    <input type="text" placeholder="Search.." name="Search">
-    <button type="submit" class="btn btn-success btn-lg"><i class="fa fa-search"></i></button>
-</div>
+  <div class=" d-flex justify-content-center">
+    <input
+      type="text"
+      placeholder="Search.."
+      name="Search"
+      [(ngModel)]="searchText"
+    />
+    <button
+      type="submit"
+      class="btn btn-success btn-lg"
+      (click)="searchVal($event)"
```

Comment:
We don't handle this event argument in "searchVal". To avoid misunderstanding and build errors we don't pass arguments if we don't need them

Reply:
Ok. Removed